diff --git a/node_modules/app-builder-lib/out/node-module-collector/moduleManager.js b/node_modules/app-builder-lib/out/node-module-collector/moduleManager.js
index d23a542..f569235 100644
--- a/node_modules/app-builder-lib/out/node-module-collector/moduleManager.js
+++ b/node_modules/app-builder-lib/out/node-module-collector/moduleManager.js
@@ -15,11 +15,11 @@ var LogMessageByKey;
     LogMessageByKey["PKG_COLLECTOR_OUTPUT"] = "collector stderr output";
 })(LogMessageByKey || (exports.LogMessageByKey = LogMessageByKey = {}));
 exports.logMessageLevelByKey = {
-    [LogMessageByKey.PKG_DUPLICATE_REF]: "info",
+    [LogMessageByKey.PKG_DUPLICATE_REF]: "debug",
     [LogMessageByKey.PKG_NOT_FOUND]: "warn",
     [LogMessageByKey.PKG_NOT_ON_DISK]: "warn",
     [LogMessageByKey.PKG_SELF_REF]: "debug",
-    [LogMessageByKey.PKG_OPTIONAL_NOT_INSTALLED]: "info",
+    [LogMessageByKey.PKG_OPTIONAL_NOT_INSTALLED]: "debug",
     [LogMessageByKey.PKG_COLLECTOR_OUTPUT]: "warn",
 };
 class ModuleManager {
